pkg/pushsync/pushsync.go

```diff
@@ -608,7 +581,7 @@ func (ps *PushSync) pushToNeighbour(ctx context.Context, peer swarm.Address, ch
 	// cancelled, then glue the stuff on the new context
 	span := tracing.FromContext(ctx)
 
-	ctx, cancel := context.WithTimeout(context.Background(), replicationTTL)
+	ctx, cancel := context.WithTimeout(context.Background(), defaultTTL)
```

Comment:
Should ctx use parent context provided via arguments?

Reply:
the neighborhood replication is has to be done independent of everything